app/models/chargeback.rb

```diff
@@ -80,7 +80,7 @@ def get_rates(perf)
     parents = [perf.parent_host, perf.parent_ems_cluster, perf.parent_storage, perf.parent_ems, @enterprise].compact
     parents.push(tenant_resource) unless tenant_resource.nil?
 
-    @rates[key] = ChargebackRate.get_assigned_for_target(perf.resource, :tag_list => tag_list, :parents => parents, :associations_preloaded => true)
+    @rates[key] = ChargebackRate.get_assigned_for_target(perf.resource, :tag_list => tag_list, :parents => parents)
```

Comment:
@kbrock Does this mean we can remove any preloading up above?  Not necessarily for this PR, but trying to understand if there's more to go.